### PR TITLE
Add Community example by denymosh

### PR DESCRIPTION
**Pull request by denymosh**
How would you rate the usage of the npx create-nillion-app process
Give a rating / 10
10/10

What platform you run your CLI on
Mac / Windows / Linux
Linux

Did you get stuck during the CLI process?
Did you experience any issues while using the latest CLI and were you able to use the template?
No Issues on setup

What is your Github repository link
Include the repository you worked on.
https://github.com/denymosh/secret_addition

ETH Address (Optional)
Add your ETH address or ENS name
0x73BbBc86104Cc3450DA89e9c83f59b3f5aEb6b22

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Nextjs Example for Create Nillion App (CNA)
+[Example Demo](https://github.com/denymosh/secret_addition)
```
Comment (crypblizz8) [inline readme.md:2]: This update is not necessary

**Comment by crypblizz8** [comment]
Please fix the other comment + then good to merge!

**Comment by denymosh** [comment]
please check it, thanks

**Comment by crypblizz8** [comment]
Great job !

**Pull request merged**